Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+def PROJECT_NAME=JOB_NAME.split("\\s|/")[2].toLowerCase()
+
+pipeline {
+    agent { 
+        docker {
+            image "fluidity/baseimages:${PROJECT_NAME}"
+            label 'azure-linux'
+        } 
+    }
+    environment {
+        MPLBACKEND = 'PS'
+    }
+    stages {
+        stage('Configuring') {   
+            steps { 
+                sh './configure --enable-2d-adaptivity' 
+            }
+        }    
+        stage('Building') {       
+            steps { 
+                sh 'make -j' ;
+                sh 'make -j fltools' ;
+                sh 'make manual'
+            }
+        }
+        stage('Testing') {       
+            steps { 
+                sh 'make -j 8 unittest' ;
```

Comment:
I would prefer not to have the `-j 8` in there: I don't think it does anything (as make only sees a single target to be built) - so it's harmless, but not something we want to suggest others to use.